src/context.ts

```diff
@@ -37,7 +37,7 @@ type WithoutCallSignature<T> = { [K in keyof T]: T[K] }
 /**
  * @deprecated Use `createStore` and `useStore` for context usage
  */
-function createContext<S extends StoreApi<unknown>>() {
+export function createContext<S extends StoreApi<unknown>>() {
```

Comment:
This is a temporary change in order to keep this PR small that will be removed in #2235

Reply:
nice idea.